docs/runbooks/PromscaleIngestHighErrorRate.md

```diff
@@ -0,0 +1,31 @@
+# PromscaleIngestHighErrorRate
+## Meaning
+Promscale is experiencing high number of errors while ingesting metric samples or traces
+
+## Impact
+For metric data, high error rates will affect Prometheus remote-storage component, leading to frequent
+retrying of samples batch and buffering of samples in the WAL, increasing Prometheus disk usage.
+If left untreated, this will lead to upsharding, increasing Prometheus memory usage. Longer retry
+durations may result dropping of samples as well.
+
+For tracing data this may result in spans being dropped.
+
+High error rates may also indicate an unhealthy database or a lack of any disk space.
+
+## Diagnosis
+1. Check the Promscale logs for errors. If they exists, go to **Ingestion data is invalid or corrupt** for mitigation
```

Comment:
```suggestion
1. Check the Promscale logs for errors. If they exists, go to [Ingestion data is invalid or corrupt](#ingestion-data-is-invalid-or-corrupt) for mitigation
```

Let's convert all go-to statements into clickable links pointing to the location where user should go.